actionlist/app/reducers/action_items.js

```diff
@@ -1,12 +1,23 @@
 /* eslint arrow-body-style: ["error", "as-needed", { "requireReturnForObjectLiteral": true }] */
 import { handleActions } from 'redux-actions';
-import { TOGGLE_DRAWER, SORT_ACTION_LIST } from '../actions/types';
+import { TOGGLE_DRAWER, SORT_ACTION_LIST, SELECT_DROPDOWN_OPTION } from '../actions/types';
 import { processInstances, sortTypes } from '../lib/commons';
 
 export const defaultState = {
   dataSource: processInstances,
   drawerExpanded: false,
   optionSelected: '',
+  dropdownColors: {
+    saved: 'white',
```

Comment:
Instead of setting these values as literals, why not make `OPTIONS` an  exported enum object and then populate the color options with `OPTIONS.values()` and then you could have, e.g., `saved: OPTIONS.WHITE,` here.